lms/templates/navigation.html

```diff
@@ -166,3 +184,6 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 % endif
 
 <%include file="help_modal.html"/>
+<%block name="js_extra">
+<%static:js group='language_edx'/>
```

Comment:
It seems a shame to add another JS file to download to every page. Can we add it to a pre-existing bundle since it is small and being included on almost every page?